packages/network-controller/tests/provider-api-tests/helpers.ts

```diff
@@ -1,17 +1,14 @@
-/* eslint-disable node/no-process-env */
```

Comment:
This file should now match the same one in the extension (minus some differences in imports, because the list of network types is defined in an enum that comes from a different place and has a different name).